images/sk-opts.proto

```diff
@@ -26,13 +26,10 @@ message sk_opts_entry {
 	optional bool		so_reuseport	= 17;
 	optional bool		so_broadcast	= 18;
 	optional bool		so_keepalive	= 19;
-	optional uint32		tcp_keepcnt	= 20;
-	optional uint32		tcp_keepidle	= 21;
-	optional uint32		tcp_keepintvl	= 22;
-	optional uint32		so_oobinline	= 23;
-	optional uint32		so_linger	= 24;
+	optional uint32		so_oobinline	= 20;
```

Comment:
This is fundamentally wrong. Image format between old and new criu should be backward-(and if possible also forward-)compatible:
1) we can't change id of the existing field
2) we can't reuse id of previously existing field.

Basically you should just mark fields 20-22 as deprecated with a comment and leave them there.

Reply:
This PR is obsolete and the latest revision is at https://github.com/checkpoint-restore/criu/pull/2380.